feat: add patient, disease information, and sleeping details forms

Rashmitharaka1 opened the pull request:
Added three standalone forms: Patient Details, Disease Information, and Sleeping Details. Centered the layout and adjusted button width to span all forms.

ThashmikaRathnayake commented on frontend/src/components/AddPatientForm.jsx:
```diff
@@ -0,0 +1,73 @@
+      <FormSection title="Sleeping Details" fields={sleepingDetails} />
+    </div>
+    <div className="flex justify-center mt-4 mb-2">
+      <button className="bg-blue-500 text-white px-6 py-2 rounded hover:bg-blue-600 transition w-full max-w-4xl cursor-pointer">
+        Add Patient
+      </button>
```
Colour should be [#34A8DD] and Validate name, symptoms and disease description as string

Rashmitharaka1 commented:
✅ Fix Implemented

Added validation to ensure only letters are allowed in Name, Symptoms, and Disease Description fields.

Blocked number input during typing using regex.

Displayed error messages when invalid input is detected.

Updated the Add Patient button color to #34A8DD.
